hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsOutputStream.java

```diff
@@ -113,7 +120,8 @@ public AbfsOutputStream(
           final Statistics statistics,
           final String path,
           final long position,
-          AbfsOutputStreamContext abfsOutputStreamContext) {
+          final Map<SelfRenewingLease, Object> leaseRefs,
```

Comment:
this should go into the AbfsOutputStreamContext, its where we are adding more state for a stream constructor